src/assets/custom/css/_sass/_services-page-intro.scss

```diff
@@ -1,23 +1,140 @@
+$header-background-nav-height--large-and-extra-large: $vertical-overlap * 4;
+$header-background-nav-image-height--large-and-extra-large: $vertical-overlap * 5;
+$header-background-nav-image-height--medium: 300px;
+$text-container-inner-margin-top: -$vertical-overlap * 2;
+
 .services-page-intro__image {
   background-color: #474b4a;
   background-position: center center;
   background-size: cover;
 
   @include small-and-medium {
-    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-small-medium.jpg");
-    height: 0;
-    padding-top: 200 / 360 * 100%;
+    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-small-medium-v2.jpg");  
   }
+
   @include large-and-extra-large {
-    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-large-extra-large.jpg");
-  }
-  @include large {
-    height: 0;
-    padding-top: 650 / 1820 * 100%;
+    background-image: url("#{$site-base-url}/assets/custom/img/services/serviceline-page-intro-large-extra-large-v2.jpg");
   }
+
   @include extra-large {
-    @include max-width--framed;
-    height: 465px;
+    height: $header-background-nav-image-height--large-and-extra-large;
+    margin-bottom: $vertical-overlap;
+  }
+}
+
+.services-page-intro__navigation {
+  @include max-width--out-there;
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Just being picky - this could potentially be scoped to the breakpoint a wrap is required.